crates/mun_syntax/src/ast/extensions.rs

```diff
@@ -210,3 +210,24 @@ impl ast::UseTree {
             .any(|it| it.kind() == T![*])
     }
 }
+
+impl ast::TypeAliasDef {
+    pub fn signature_range(&self) -> TextRange {
```

Comment:
A comment might help here. What do you mean by "signature" range?

Reply:
Added to all implementations (typealias, struct, fn)